### ISSUE TITLE
Images in template tags

### ISSUE THREAD

**Issue by jdiamond** [TylorS/typed-content-hash#11]
Hi, I'm using snowpack-plugin-hash. When an img tag is inside a template tag, the image file (.svg in my case) does not get the hash added. When I copy the img tag outside of the template tag and rebuild, the hash does get added to the file name.

**Comment by TylorS** [comment]
👋 Hey @jdiamond, thanks for reaching out! I'm sorry to hear things aren't working quite as expected. 

Would you be able to create a minimal reproduction case for this so I can make sure I'm tackling the right thing?

If possible, it'd be great to have a case added to the "project" inside the `/test` folder if you'd be willing to make PR? Then I could debug and get a fix out for it relatively quickly and ensure it doesn't happen again 😄

**Comment by TylorS** [comment]
Hey @jdiamond, I was able to add a test case for this and put [together a fix](https://github.com/TylorS/typed-content-hash/commit/73a16295a492535e4478a6d142f92f7872c0bcdd). This has been released in 0.10.0, and I'll get an update together for `snowpack-plugin-hash`

**Comment by TylorS** [comment]
Feel free to reopen if this fix didn't hit the mark

**Comment by jdiamond** [comment]
Hi, thanks for looking into this. It was no big deal for me to change the template tags to divs in my project as an easy workaround. I just tried the previous version I was using (0.14.2) of snowpack-plugin-hash on a new project with an img inside a template and the image did not get hashed, then I upgraded to the latest (0.16.0) and now it works so I think you got it. Your plugin works really well with the built-in optimizer in snowpack. Thanks for making it available.
